src/fileSystemActions.test.js

```diff
@@ -11,46 +11,49 @@ describe("saveToFile", () => {
     jest.resetAllMocks();
   });
 
-  it("saves content to a file", () => {
+  it("saves content to a file", async () => {
```

Comment:
This is the test I'm focusing on. 

- `nativeUI.getFileNameFromUser()` is stubbed to return a resolved promise
- `fs.writeFile()` is stubbed to return a resolved promise.
  - This is what I don't think is working...
- I `await` for the method to finish it's async operations
- I expect `nativeUI.displayInfoMessage()` to be called <-- this fails
- I expect `fs.writeFile()` to be called <-- this passes